scripts/firedrake-configure

```diff
@@ -191,6 +191,22 @@ PETSC_EXTRAS_LINUX_APT_PACKAGES = (
 
 COMMON_LINUX_APT_PACKAGES = BASE_LINUX_APT_PACKAGES + PETSC_EXTRAS_LINUX_APT_PACKAGES
 
+LINUX_X86_64_APT_PACKAGES = (
+    "libmumps-ptscotch-dev",
+    "libmetis-dev",
+    "libnetcdf-dev",
+    "libpnetcdf-dev",
+    "libptscotch-dev",
+    "libscalapack-openmpi-dev",
+    "libsuitesparse-dev",
+    "libsuperlu-dev",
+    "libsuperlu-dist-dev",
+)
+LINUX_AARCH64_APT_PACKAGES = LINUX_X86_64_APT_PACKAGES
```

Comment:
If these are truly identical they should probably go in `PETSC_EXTRAS_LINUX_APT_PACKAGES`.

Reply:
Agreed, they were just separate while I was getting things working. I've added them onto the existing package list now.